lib/ui/views/course_landing/widgets/course_landing_overview.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:filledstacked_academy/ui/common/app_colors.dart';
+import 'package:filledstacked_academy/ui/common/shared_styles.dart';
+import 'package:filledstacked_academy/ui/common/ui_helpers.dart';
+import 'package:filledstacked_academy/ui/widgets/common/academy_button.dart';
+import 'package:filledstacked_academy/ui/widgets/common/course_benefits_card.dart';
+import 'package:filledstacked_academy/ui/widgets/common/course_price_card.dart';
+import 'package:flutter/material.dart';
+
+class CourseLandingOverview extends StatelessWidget {
+  const CourseLandingOverview({Key? key}) : super(key: key);
+
+  static const String courseDetails = '''
+The course teaches patterns, file layouts and development principles used to build the FilledStacks Academy. Yes, the website you’re using right now. 
```

Comment:
This we should get from `course.description`

Reply:
I forgot to remove the hardcoding data 😅 . Done ✅ .